udev/revpi_mac

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+DEVNAME=$1
+OFFSET=$2
+
+read -r BASE_MAC < /proc/device-tree/hat/custom_5
+if [ -z "$BASE_MAC" ]; then
+    exit 1
+fi
+
+MAC_NUM=$(echo $BASE_MAC | sed -e 's/://g')
+OFFSET_NUM=$(printf "%x" $((16#$MAC_NUM + $OFFSET)))
+OFFSET_MAC=$(echo $OFFSET_NUM | sed 's/.\{2\}/&:/g')
```

Comment:
```suggestion
OFFSET_MAC=$(echo "$OFFSET_NUM" | sed 's/.\{2\}/&:/g')
OFFSET_MAC=${OFFSET_MAC::-1}
```

SC2086 & better readability (second line; at least for my taste, feel free to ignore if you don't agree)